Calculator.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FirstProject
+{
+    internal class Calculator
+    {
+        
+        public int add()
+        {
+            Console.WriteLine("Podaj dwie liczby");
+            int a = Convert.ToInt32(Console.ReadLine());
+            int b = Convert.ToInt32(Console.ReadLine());
+            Console.ForegroundColor = ConsoleColor.Green;
```

Comment:
Nie resetujemy nigdzie stanu? Co się stanie jak będę chciał wykonać 2 operację pod tym kodem?

Reply:
Wstępnie zrobione tylko dla dodawania. Niestety to jedyne rozwiązanie które udało mi się stworzyć i w miare jako tako działa :)